var/spack/repos/builtin/packages/py-pyside/package.py

```diff
@@ -42,6 +42,17 @@ def patch(self):
         rpath.append(os.path.join(
             self.prefix, pypkg.site_packages_dir, 'PySide'))
 
+        # Fix subprocess.mswindows check for Python 3.5
+        # https://github.com/pyside/pyside-setup/pull/55
+        filter_file(
+            r'^if subprocess.mswindows:',
+            'mswindows = (sys.platform == "win32")\r\nif mswindows:',
```

Comment:
The file uses Windows line endings?

Reply:
> The file uses Windows line endings?

I'm just trying to stay consistent with the rest of the file (see line 81).